src/pmpo_materialPoints.cpp

```diff
@@ -2,6 +2,19 @@
 
 namespace polyMPO {
 
+pumipic::MemberTypeViews createInternalMemberViews(int numMPs, IntView mp2elm, IntView mpAppID){
+  auto mpInfo = ps::createMemberViews<MaterialPointTypes>(numMPs);
+  auto mpCurElmPos_m = ps::getMemberView<MaterialPointTypes, MPF_Cur_Elm_ID>(mpInfo);
+  auto mpAppID_m = ps::getMemberView<MaterialPointTypes, MPF_MP_APP_ID>(mpInfo);
+  auto mpStatus_m = ps::getMemberView<MaterialPointTypes, MPF_Status>(mpInfo);
+  Kokkos::parallel_for("setMPinfo", numMPs, KOKKOS_LAMBDA(int i) {
+    mpCurElmPos_m(i) = mp2elm(i);
+    mpStatus_m(i) = 1;
```

Comment:
For consistency, do we want to use something like `MP_ACTIVE` instead of `1` here?

Reply:
I don't see why not. I made the change.